packages/gatsby-plugin-typedoc/src/lib/createCommentNode.ts

```diff
@@ -0,0 +1,80 @@
+const commentId = (parentId, type) => `${parentId}--${type}TypedocCommentValue`;
+
+const toTextNodeFactory = ({
+  createNode,
+  createNodeId,
+  createContentDigest,
+  id
+}) => (value, type: 'Text' | 'ShortText') => {
+  const node = {
+    value,
+    id: createNodeId(commentId(id, type)),
+    parent: id,
+    children: [],
+    internal: {
+      type: `TypedocCommentValue`,
+      mediaType: `text/markdown`,
+      content: value,
+      contentDigest: createContentDigest(value)
+    }
+  };
+
+  createNode(node);
+  return node;
+};
+
+interface CommentNode {
+  id: string;
+  parent: string;
+  children: string[];
+  text?: string;
+  shortText?: string;
+  tags?: any;
+}
+
+function createCommentNode(
+  node,
+  entry: { tags?: string[]; text?: string; shortText?: string },
+  actions,
+  createNodeId,
+  createContentDigest
+) {
+  const { createNode } = actions;
+
+  const toTextNode = toTextNodeFactory({
+    createNode,
+    createNodeId,
+    createContentDigest,
+    id: node.id
+  });
+
+  const commentNode: CommentNode = {
+    id: createNodeId(commentId(node.id, '')),
+    parent: node.id,
```

Comment:
you also need an `internal` field here, every node (not graphql Node, gatsby Node) needs it